applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -331,7 +331,6 @@ config BT_AUDIO_BROADCAST_ENCRYPTED
 	default n
 	help
 	  Encrypt the broadcast to limit the connectability
```

```suggestion
	  Encrypt the broadcast to limit the connection possibilities.
```
"Connectability" is not present in any dictionary. Use proposed or "connectivity".